spec/jobs/jira_defect_metrics_updater_job_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe JiraDefectMetricsUpdaterJob do
```

Comment:
```suggestion
describe JiraDefectMetricsUpdaterJob do
```

I think this applies to all `_spec` files